pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -335,9 +336,10 @@ void writeMetadata()
     ColumnIndexCreationInfo timeColumnIndexCreationInfo = indexCreationInfoMap.get(timeColumn);
     if (timeColumnIndexCreationInfo != null) {
       // Use start/end time in config if defined
```

Comment:
Please update the comment here if logic changes.

Reply:
The logic is not changed, but a bug fix. Updated the comments to make that more clear